diff/diff.py

```diff
@@ -0,0 +1,81 @@
+import os
+import json
+import logging
+
+from deepdiff import DeepDiff
+from diff.moosedeepdiff import MOOSEDeepDiff
+
+
+def compare_jsons(json_file01, json_file02, relative_error=None, absolute_error=None):
+    """Determine whether there is a difference between two json files
+        relative_error: the maximum relative tolerance that will be detected as a changed value
+        absolute_error: the maximum absolute tolerance that will be detected as a changed value
+        Return return code (int) if an error occurred
+        Return deep diff object otherwise 
+            e.g. differences: {'values_changed': {'root[0]': {'new_value': 1.0000001, 'old_value': 1.0}}} 
+            e.g. no differences: {}
+        Formatted string of the differences: deep_diff_obj.pretty()
+        Note: Must pip install and import deepdiff"""
+    return_code = 0
+    
+    # Validate files
+    return_code += validate_tolerance(relative_error, absolute_error)
+    return_code += validate_json_file(json_file01)
+    return_code += validate_json_file(json_file02)
```

Comment:
These should use the mooseutils.validate functions, and only use the logging version.